.github/workflows/release.yml

```diff
@@ -24,10 +24,13 @@ jobs:
           wget -O- https://github.com/intuit/auto/releases/download/v10.16.1/auto-linux.gz | gunzip > ~/auto
           chmod a+x ~/auto
 
-      - name: Check whether a release is due
+      - name: Query 'auto' on type of the release
         id: auto-version
         run: |
-          version="$(~/auto version)"
+          # to be able to debug if something goes wrong
+          set -o pipefail
+          auto version -v | tee /tmp/auto-version
+          version=$(sed -ne '/Calculated SEMVER bump:/s,.*: *,,p' < /tmp/auto-version)
```

Comment:
```suggestion
          version="$(sed -ne '/Calculated SEMVER bump:/s,.*: *,,p' /tmp/auto-version)"
```

Reply:
according to shellcheck/trial it is not needed

```
❯ shellcheck /tmp/1.sh
❯ /tmp/1.sh
patch with space
❯ cat /tmp/1.sh
#!/bin/sh

version=$(sed -ne '/Calculated SEMVER bump:/s,.*: *,,p' /tmp/auto-version)
echo "$version"
```